Formula/podman.rb

```diff
@@ -36,6 +36,8 @@ def install
     end
     on_linux do
       bin.install "bin/podman-remote"
+      system "make", "podman"
+      bin.install "bin/podman"
```

Comment:
adding this after following the discussion from https://github.com/Homebrew/homebrew-core/pull/83578#issuecomment-902126237,